packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -93,6 +93,11 @@ export interface IOdspErrorAugmentations {
  */
 export interface IOdspError extends Omit<IDriverErrorBase, "errorType">, IOdspErrorAugmentations {
 	readonly errorType: OdspErrorTypes;
+	/**
+	 * In case of socket io errors, this property captures the socket event which caused the error
+	 * i.e. connect_error, connect_document_error etc.
+	 */
+	errorFrom?: string;
```

Comment:
I don't like the name "errorFrom" as it is too braod and as per your comment it does not mention what it will be in case it is not from socket. Should we just add it to IAnyDriverError and remove it from IOdsperror as there is nothing odsp specific in "errorFrom" as we added it to r11s driver also.

Reply:
No changes have been made in the r11s as I don't wish to modify code in unknown territory just yet. For now, this PR only touches that code to perform a variable rename.